src/py/reactpy/reactpy/backend/default.py

```diff
@@ -59,7 +60,10 @@ def _default_implementation() -> BackendImplementation[Any]:
         implementation = next(all_implementations())
     except StopIteration:  # nocov
         logger.debug("Backend implementation import failed", exc_info=exc_info())
-        msg = "No built-in server implementation installed."
+        supported_backends = ', '.join(SUPPORTED_PACKAGES)
+        msg = "It seems you haven't installed any built-in backends. \n" \
+            "To resolve this issue, you can install a backend like 'reactpy[starlette]' or any other supported backend.\n" \
+            f"The list of supported backends is: {supported_backends}."     
```

Comment:
I'd like to get the exact `pip install` command in there so the user can just copy-paste it to fix the problem. So maybe something worded/formatted like this:

```
It seems like you haven't installed a backend. You can do so by running:

pip install "reactpy[starlette]"

Other supported backends include: {supported_backends}
```